examples/binary-operators/binary_operators.md

```diff
@@ -0,0 +1,9 @@
+# Binary operators
+
+Ballerina provides several binary operators to perform operations on values of various types such as integers, floats, booleans, and strings. Binary arithmetic operators (`+`, `-`, `*`, `/`, `%`) allow basic mathematical calculations between two values. The operators to compare values (`<`, `>`, `<=`, `>=`) produce a boolean result.
```

Comment:
Fix operator lists to use ", and"

Reply:
Doesn't cover + for string.

Reply:
Since it is describe in the example, Ino no need to add in here